src/Umbraco.Cms.Integrations.Crm.ActiveCampaign/App_Plugins/UmbracoCms.Integrations/Crm/ActiveCampaign/js/formpicker.controller.js

```diff
@@ -0,0 +1,93 @@
+﻿function formPickerController($scope, editorService, notificationsService, umbracoCmsIntegrationsCrmActiveCampaignResource) {
+
+    var vm = this;
+
+    vm.loading = false;
+    vm.searchTerm = "";
+
+    vm.selectedForm = {};
+
+    vm.status = "";
+
+    umbracoCmsIntegrationsCrmActiveCampaignResource.checkApiAccess().then(function (response) {
+        vm.isApiConfigurationValid = response.isApiConfigurationValid;
+        if (response.isApiConfigurationValid) {
+            if ($scope.model.value) {
+                getFormDetails($scope.model.value);
+            }
+
+            loadForms();
+        }
+        else {
+            vm.status = "Invalid API configuration.";
+        }
+    });
+
+    vm.saveForm = function (formId) {
+        $scope.model.value = formId;
+
+        getFormDetails(formId);
+    }
+
+    vm.removeForm = function () {
+        $scope.model.value = null;
+        vm.selectedForm = {};
+    }
+
+    vm.openActiveCampaignFormPickerOverlay = function () {
+
+        var options = {
+            title: "ActiveCampaign Forms",
+            subtitle: "Select a form",
+            view: "/App_Plugins/UmbracoCms.Integrations/Crm/ActiveCampaign/views/formpickereditor.html",
+            size: "medium",
+            selectForm: function (form) {
+
+                if (form.id !== undefined) {
+                    vm.saveForm(form.id);
+                }
+
+                editorService.close();
+            },
+            close: function () {
+                editorService.close();
+            }
+        };
+
+        editorService.open(options);
+    };
+
+    function getFormDetails(id) {
+        vm.loading = true;
+        umbracoCmsIntegrationsCrmActiveCampaignResource.getForm(id).then(function (response) {
+            if (response.message !== null && response.message.length > 0)
```

Comment:
```suggestion
            if (response.message && response.message.length > 0)
```

This way you deal with undefined as well as null.